### ISSUE TITLE
Level Completion

### ISSUE THREAD

**Issue by nsmryan** [nsmryan/RustRoguelike#403]
Some ideas of ways that the player could finish a level:

reach an exit tile
reach the exit tile with a key
press a trigger to unlock the exit, perhaps requiring the trigger to be down with an item or golem
reach an edge of the level
reach multiple tiles, such as triggers, before the exit opens
a vault layout with a golem blocking the exit

**Comment by MicroChasm** [comment]
I like the idea that you have to press a pressure plate or something.  The idea of having a golem in the way is kind of cool too, since it would mean that you can't just run. 

For now, for the sake of testing, I think it would be nice to have a very straightforward objective, so I think that reaching a tile on the rightmost side would be enough.

**Comment by MicroChasm** [comment]
I think that the game works best if the player has a reasonable idea of where to go, since I can see it being very frustrating to wander the level and use all your resources searching for a key. Maybe we could come up with some reason that the player simply knows where the key is at all times, or something like that.

**Comment by nsmryan** [comment]
Agreed on both account: better to keep it simple for now, and make sure the player knows what they are doing.
I just wanted to track some ideas for when we get to level design

**Comment by MicroChasm** [comment]
I wanted to include the idea in this issue that the levels should be elongated a bit.
